wikichron/dash/networks/models/CoEditingNetwork.py

```diff
@@ -42,13 +42,13 @@ class CoEditingNetwork(BaseNetwork):
     AVAILABLE_METRICS = {
             'Page Rank': 'page_rank',
             'Number of Edits': 'num_edits',
-            'Betweenness': 'betweenness'
+            'Betweenness': 'betweenness',
+            'Cluster': 'cluster'
         }
 
     USER_INFO = {
-        'Wiki ID': 'contributor_id',
+        'Wiki ID': 'id',

```

Comment:
Wiki ID? shouldn't it be User id?

Reply:
Let it be, may be I'll remove this attr from USER_INFO